readme.md

```diff
@@ -80,6 +80,7 @@ Click the type names for complete docs.
 - [`ConditionalPick`](source/conditional-pick.d.ts) - Like `Pick` except it selects properties from a shape where the values extend the given `Condition` type.
 - [`ConditionalExcept`](source/conditional-except.d.ts) - Like `Omit` except it removes properties from a shape where the values extend the given `Condition` type.
 - [`UnionToIntersection`](source/union-to-intersection.d.ts) - Convert a union type to an intersection type.
+- [`UnionToTuple`](source/union-to-tuple.d.ts) - Convert an explicit union type to a tuple.
```

Comment:
`UnionToIntersection` says `Convert a union type to an intersection type.`

I would expect this one to say something like:

```suggestion
- [`UnionToIntersection`](source/union-to-intersection.d.ts) - Convert a union type to an intersection type.
- [`UnionToTuple`](source/union-to-tuple.d.ts) - Convert a user defined union type to a tuple type. Order can not be guaranteed on computed union types.
```

Reply:
I think this is clearer.